pkg/controller/replication_controller_test.go

```diff
@@ -183,10 +212,11 @@ func TestSyncReplicationControllerDeletes(t *testing.T) {
 	fakePodControl := FakePodControl{}
 
 	manager := NewReplicationManager(client)
+	close(manager.stopCh)
 	manager.podControl = &fakePodControl
+	manager.podLister = newStoreToPodLister(podList, cache.NewStore(cache.MetaNamespaceKeyFunc))
```

Comment:
I believe that in most (all?) tests you have identical sequence:

client := client.NewOrDie(&client.Config{Host: testServer.URL, Version: testapi.Version()})
manager := NewReplicationManager(client)
close(manager.stopCh)
manager.podControl = &fakePodControl
manager.podLister = newStoreToPodLister(podList, cache.NewStore(cache.MetaNamespaceKeyFunc))

How about adding a helper function to wrap it?

manager, fakePodControl := newTestReplicationManager(testServer, podList)